File: Excercise_Files/04_javaScript/07_multiplicationTable.js
// JS program that gives you the 10 times multiplication table of a chosen number

// user input
let base = Number(prompt('Which whole number do you choose for your multiplication table? '));

// a for loop to do the multiplication 10 times
// variable i will also act as the multiplier
// print it all on screen
for (let i = 1; i <= 10; i++) {
  console.log(base + ' x ' + i + ' = ' + base * i);
}

File: Excercise_Files/02_Python/07_multiplicationTable.py
# simple program that gives you the 10 times multiplication table of a chosen number

# user input
base = int(input('Which whole number do you choose for your multiplication table? '))

# a while loop will do the multiplication 10 times
# fortunately on the way we can use 'i' from the condition to multiply our chosen number with
i = 1
while i < 11:
  print(base, 'x', i, '=', base * i)
  i += 1
